react/features/base/react/components/native/LoadingIndicator.js

```diff
@@ -27,12 +27,25 @@ export default class LoadingIndicator extends Component<Props> {
      * @returns {ReactElement}
      */
     render() {
-        return (
+        let { size = 'large' } = this.props;
+
+        if (size === 'medium') {
+            size = 'large';
```

Comment:
where is this translation coming from ?

Reply:
The web component - the atlaskit spinner.

Reply:
@paweldomas With this commit I've added a LoadingIndicator for web which is basically the atlaskit's spiner component. My motivation to do this was to use the exact same JS code for the StartRecording dialog for web and native. The possible values for the size prop of atlaskit's spiner component are - small, medium and large. In the same time the react-native's ActivityIndicator component supports only small and large. And I want to have the exact same API for LoadingIndicator on web and native. That's why when you pass 'medium' on native I need to translate it to a value supported by the react-native's ActivityIndicator component.

Reply:
That’s cool. All I’m asking is to put a comment: ‘The native “ActivityIndicator” component supports only “small” and “large” sizes.’
Otherwise I’m scratching my head if it’s some hack to never display it as medium. It’s just not clear enough